src/item.cpp

```diff
@@ -549,7 +549,7 @@ std::string item::info( bool showtext, std::vector<iteminfo> &info ) const
                                   convert_weight( weight() ), false, "", true, true ) );
 
         if( count_by_charges() ) {
-            info.emplace_back( "BASE", _( "Stack size: " ), "", type->stack_size, true );
+            info.emplace_back( "BASE", _( "Stack size: " ), "", type->stack_size / std::min( type->volume, 1U ) , true );
```

Comment:
What has the "Stack size" information to do with the volume? Isn't it supposed to show the stack size itself, as in "the number of charges per default stack"? If the calculation is the desired thing to show, the information should be renamed to something like "Charges per 1 volume" or so.

Also: `type->volume` can be 0, can't it? If so, `std::min( type->volume, 1U )` can easily be 0 as well.